src/service/video-manager-impl.js

```diff
@@ -459,10 +459,7 @@ class VideoEntry {
 
     // eslint-disable-next-line jsdoc/require-returns
     /** @private @const {function(): !Promise<boolean>} */
-    this.supportsAutoplay_ = () => {
-      const {win} = this.ampdoc_;
-      return VideoUtils.isAutoplaySupported(win, getMode(win).lite);
-    };
+    this.isAutoplaySupported_ = () => VideoUtils.isAutoplaySupported(this.win);
```

Comment:
nit: is this a useful variable? the alternative being to call `import {isAutoplaySupported} from 'src/utils/video'`;

Reply:
`VideoUtils` is actually a class containing static methods. It was added before our test builds could stub module exports. I'll follow up with a PR that simplifies the autoplay support module.